scripts/patches/fb2k_utils.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/submodules/fb2k_utils/src/qwr/qwr_exception.h b/submodules/fb2k_utils/src/qwr/qwr_exception.h
+index 7e75b04..de7a01c 100644
+--- a/submodules/fb2k_utils/src/qwr/qwr_exception.h
++++ b/submodules/fb2k_utils/src/qwr/qwr_exception.h
+@@ -12,7 +12,7 @@ class QwrException
+ public:
+     template <typename... Args>
+     explicit QwrException( qwr::u8string_view errorMessage, Args&&... errorMessageFmtArgs )
+-        : std::runtime_error( fmt::format( errorMessage, std::forward<Args>( errorMessageFmtArgs )... ) )
++        : std::runtime_error( fmt::format( fmt::runtime( errorMessage ), std::forward<Args>( errorMessageFmtArgs )... ) )
```

Comment:
well, it's failing compilation for me on this callback in just about every invocation (on msvc2022/v143)  relating to fmt v8's enforced compile time format string checks

<details>
<summary>error message example</summary>
<pre>
foo_spider_monkey_panel\submodules\fb2k_utils\src\qwr\qwr_exception.h(15,44): error C7595: 'fmt::v8::basic_format_string<char>::basic_format_string': call to immediate function is not a constant expression (compiling source file utils\menu_helpers.cpp)
foo_spider_monkey_panel\submodules\fmt\include\fmt\core.h(3057,67): message : failure was caused by a read of a variable outside its lifetime (compiling source file utils\menu_helpers.cpp)
foo_spider_monkey_panel\submodules\fmt\include\fmt\core.h(3057,67): message : see usage of 'errorMessage' (compiling source file utils\menu_helpers.cpp)
foo_spider_monkey_panel\foo_spider_monkey_panel\utils\menu_helpers.cpp(266): message : see reference to function template instantiation 'qwr::QwrException::QwrException<>(qwr::u8string_view)' being compiled
(... repeated ... )
</pre>
</details>

and wrapping `errorMessage` in fmt::runtime seemed like the obvious (easy/lower impact) solution versus going in and adjusting every call.

Reply:
interesting, perhaps it's a v143 compiler change (or a bug)